src/mscorlib/src/System/Resources/LooselyLinkedResourceReference.cs

```diff
@@ -27,7 +27,6 @@ namespace System.Resources {
     using System.Globalization;
 
     [Serializable]
```

Comment:
I don't think it gets into core as the it contains a conditional compilation: #if LOOSELY_LINKED_RESOURCE_REFERENCE. Should I still remove it?